src/lib/common/topology_sampler.cpp

```diff
@@ -149,12 +199,60 @@ static void topology_timer_cb(void *arg) {
  * @return none
  */
 static void topology_timer_handler(TimerHandle_t tmr) {
-    (void) tmr;
-    timer_callback_handler_send_cb(topology_timer_cb, NULL, 0);
+  (void)tmr;
+  timer_callback_handler_send_cb(topology_timer_cb, NULL, 0);
 }
 
+/**
+ * @brief  Callback function for sys info reply
+ * 
+ * @note Computes a node's crc for aggregation into the network crc
+ * 
+ * @param[in] ack True if the message was acknowledged, false otherwise.
+ * @param[in] msg_id The message id.
+ * @param[in] service_strlen The length of the service string.
+ * @param[in] service The service string.
+ * @param[in] reply_len The length of the reply data.
+ * @param[in] reply_data The reply data.
+ * @return True if the request was handled, false otherwise.
+ */
+static bool sys_info_reply_cb(bool ack, uint32_t msg_id, size_t service_strlen,
```

Comment:
Sys info callback for each of the nodes - prints out the info, computes the crc based on the info, and sends it back to the topology sampler task